opentelemetry-user-events-logs/benches/logs.rs

```diff
@@ -0,0 +1,80 @@
+/*
+    The benchmark results:
+    criterion = "0.5.1"
+
+    Hardware: Apple M4 Pro
+    Total Number of Cores:	10
+    // When no listener
+    | Test                        | Average time|
+    |-----------------------------|-------------|
+    | User_Event_4_Attributes     | 10 ns       |
+    | User_Event_6_Attributes     | 10 ns       |
+
+    // When listener is enabled
+    | Test                        | Average time|
+    |-----------------------------|-------------|
+    | User_Event_4_Attributes     | 10 ns       |
+    | User_Event_6_Attributes     | 10 ns       |
+*/
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use opentelemetry_appender_tracing::layer as tracing_layer;
+use opentelemetry_sdk::logs::SdkLoggerProvider;
+use opentelemetry_sdk::Resource;
+use opentelemetry_user_events_logs::UserEventsLoggerProviderBuilderExt;
+use tracing::error;
+use tracing_subscriber::prelude::*;
+use tracing_subscriber::Registry;
+
+fn benchmark_with_ot_layer(c: &mut Criterion, name: &str, num_attributes: usize) {
+    let provider = SdkLoggerProvider::builder()
+        .with_resource(
+            Resource::builder_empty()
+                .with_service_name("benchmark")
+                .build(),
+        )
+        .with_user_event_exporter("myprovider")
+        .build();
+    let ot_layer = tracing_layer::OpenTelemetryTracingBridge::new(&provider);
+    let subscriber = Registry::default().with(ot_layer);
+
+    tracing::subscriber::with_default(subscriber, || {
+        c.bench_function(name, |b| {
+            b.iter(|| {
+                if num_attributes == 4 {
```

Comment:
Have two separate benchmark methods instead of doing an `if` check within the measurement iteration.

Reply:
yes we can make it better in a future iteration. Just wanted to measure the cost when listener is enabled (serialization etc. cost0